src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day calendar--month-length">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--mon calendar--month-length--30">
```
тут мають бути модифікатори ,змінюючи які змінюеться кількість днів в календарі і встановлюеться день з якого починаеться місяць

Reply:
Добре, думала, що їх не потрібно прописувати якісь конкретні, в стилях вони є